website/www/site/content/en/documentation/runners/basics.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Basics of the Beam model"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
```

Comment:
This file needs to be deleted. I'll do that after merging this PR.

Reply:
Done - https://github.com/apache/beam/pull/13871/commits/67508d0ce82ac1f793e03cfcc98bc9b9dc28ccff